ee/ucp/admin/configure/join-nodes/join-linux-nodes-to-cluster.md

```diff
@@ -123,7 +123,7 @@ To remove a manager node:
 1. Make sure all nodes in the cluster are healthy. Don't remove manager nodes
```

Comment:
Suggested edit:

1. Confirm that all nodes in the cluster are healthy (otherwise, do not remove manager nodes).
2. Demote the manager nodes into workers.
3. Remove the newly-demoted workers from the cluster.